lib/app.js

```diff
@@ -24,7 +24,7 @@ app.use('/api', ensureAuth);
 // and now every request that has a token in the Authorization header will have a `req.userId` property for us to see who's talking
 app.get('/api/test', (req, res) => {
   res.json({
-    message: `in this proctected route, we get the user's id like so: ${req.userId}`
+    message: `in this protected route, we get the user's id like so: ${req.animalsId}`
```

Comment:
You can ignore these /api/ routes for now. It'll be relevant later, but for now you can leave it alone